spec/unit/pops/loaders/loaders_spec.rb

```diff
@@ -107,7 +107,7 @@
     end
 
     it 'loader allows loading a function more than once' do
-      File.stubs(:read).with(user_metadata_path).returns ''
+      Puppet::FileSystem.stubs(:read).with(user_metadata_path, {:encoding => 'utf-8'}).returns ''
```

Comment:
@Iristyle You are changing from `File` to `FileSystem` - does it work if you change that?